parserator_web/views.py

```diff
@@ -14,9 +15,25 @@ class AddressParse(APIView):
     renderer_classes = [JSONRenderer]
 
     def get(self, request):
-        return Response({})
+        # I'm passing in the input string as a request parameter
```

Comment:
can you discuss why not? (i'm not asking you to change it, just discuss your thought process)

Reply:
Sure! I was getting some unexpected errors when I used ParseError to return an error (except ParseError as e). I couldn't quite get it to work! I would be happy to try again.